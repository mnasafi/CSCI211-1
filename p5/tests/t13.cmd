2 3 tests/t13.in results/t13.myout
