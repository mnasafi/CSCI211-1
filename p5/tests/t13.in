Marge shopper 2 4
Maggie shopper 4 2
Bart shopper 3 3
Lisa robber 5 1
Homer shopper 1 5
