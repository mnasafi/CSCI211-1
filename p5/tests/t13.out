1: Homer entered store
2: Marge entered store
3: Bart entered store
4: Maggie entered store
5: Lisa entered store
7: Lisa done shopping
7: Lisa started checkout with checker 0
8: Maggie done shopping
8: Maggie started checkout with checker 1
9: Bart done shopping
10: Marge done shopping
10: Maggie paid $6 for 2 items to checker 1
10: Bart started checkout with checker 1
11: Homer done shopping
11: Lisa stole $100 and 1 item from checker 0
13: Bart paid $9 for 3 items to checker 1
13: Marge started checkout with checker 1
14: Homer started checkout with checker 0
17: Marge paid $12 for 4 items to checker 1
19: Homer paid $15 for 5 items to checker 0
registers[0] = $15
registers[1] = $127
time = 20
